README.md

```diff
@@ -74,6 +74,7 @@ Table of Contents
   * https://www.blockspring.com/ - Share scripts with anyone on your team: cross language and with spreadsheet users. Free for 5 million runs a month.
   * https://helpmonks.com/ - Shared inbox for teams - Free for open source projects and non-profit organizations.
   * http://typetalk.in/ - Share and discuss ideas with your team through instant messaging on the web or on your mobile.
+  * https://www.wunderlist.com/ - Share your lists and work collaboratively on projects with your colleagues - Available for free on iPhone,iPad,Mac,Android,Windows and the Web.
```

Comment:
it’s free to download and use , and  also offers a purchase version .  the free version would satisfy the requirement from small team completely. 

https://itunes.apple.com/app/wunderlist-to-do-list-tasks/id410628904 <https://itunes.apple.com/app/wunderlist-to-do-list-tasks/id410628904>
the description in iTunes is below : 
>Wunderlist is free to download and use. Wunderlist Pro upgrades your experience and gives you unlimited access to Files, Assigning and >Subtasks to help you accomplish even more for $4.99 a month or $49.99 a year through an in-app purchase.




Best Regards,
Frank Bian



> On May 14, 2015, at 5:59 PM, R.I.Pienaar <notifications@github.com> wrote:
> 
> In README.md <https://github.com/ripienaar/free-for-dev/pull/343#discussion_r30309891>:
> 
> > @@ -74,6 +74,7 @@ Table of Contents
> >    * https://www.blockspring.com/ - Share scripts with anyone on your team: cross language and with spreadsheet users. Free for 5 million runs a month.
> >    * https://helpmonks.com/ - Shared inbox for teams - Free for open source projects and non-profit organizations.
> >    * http://typetalk.in/ - Share and discuss ideas with your team through instant messaging on the web or on your mobile.
> > +  * https://www.wunderlist.com/ - Share your lists and work collaboratively on projects with your colleagues - Available for free on iPhone,iPad,Mac,Android,Windows and the Web.
> do you have a link that shows what the terms of the free trial is? stays free with limitations or time based? every link that I tried to get more details seem to insist on a signup
> 
> —
> Reply to this email directly or view it on GitHub <https://github.com/ripienaar/free-for-dev/pull/343/files#r30309891>.
>